tracing-futures/src/lib.rs

```diff
@@ -157,7 +191,9 @@ mod tests {
     extern crate tokio;
 
     use super::{test_support::*, *};
-    use futures::{future, stream, task, Async};
+
+    #[cfg(feature = "futures-01")]
```

Comment:
Instead of putting `cfg` attributes on each individual import and test in this module, what do you think of having separate test modules for `futures-01` and `std-future`? Then we can just feature-flag the entire module, and not have to worry about individual items within that module.

Reply:
That works for me. Both test modules would use the same base `PollN` so there will also need to be a third module that is used by both testing modules.

Reply:
The feature-flagged modules could be submodules of the main test module?

Reply:
It definitely works like that. I can make it a separate top-level module, but then I have to mark `PollN` as `pub(crate)` and it's two constructors as `pub(crate)`.